src/main/java/com/piseth/java/school/schoolManagement/controller/ScoreController.java

```diff
@@ -0,0 +1,49 @@
+package com.piseth.java.school.schoolManagement.controller;
+
+import java.util.List;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.piseth.java.school.schoolManagement.dto.MonthlyScoreDTO;
+import com.piseth.java.school.schoolManagement.dto.StudentDTO;
+import com.piseth.java.school.schoolManagement.model.MonthlyScore;
+import com.piseth.java.school.schoolManagement.service.ScoreService;
+
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequestMapping("/score/")
+@RequiredArgsConstructor
+public class ScoreController {
+	private final ScoreService scoreService ;
+	@PostMapping("/student/{id}/subject/{id1}")
```

Comment:
-if you use @PostMapping("/student/{id}/subject/{id1}") , you should provide a good name for parameter like studentId...
-Anyway, no need to use path variable, you can get the studentId and subject id from MonthlyScoreDTO.